cime_config/config_compsets.xml

```diff
@@ -85,20 +85,20 @@
   </compset>
   <compset>
     <alias>I2000Clm60SpRs</alias>
-    <lname>2000_DATM%GSWP3v1_CLM60%SP_SICE_SOCN_SROF_SGLC_SWAV</lname>
+    <lname>2000_DATM%CRUJRA2024_CLM60%SP_SICE_SOCN_SROF_SGLC_SWAV</lname>
```

Comment:
Change these back: the ones that do not explicitly specify which datm data is used. This way testing for this PR/tag will remain b4b, and we will introduce CRUJRA2024 as the default in an answer-changing PR/tag.